install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -425,11 +425,6 @@ spec:
           valueFrom:
             configMapKeyRef:
               key: clean-cilium-bpf-state
-              name: cilium-config
-              optional: true
```

Comment:
This should be kept L425-L429

Reply:
I did exactly what the original commit did, `diff` just displays it weirdly. But I removed the `key: wait-bpf-mount` line now, which means https://github.com/cilium/cilium/pull/17123 will not have to be backported.